### PR TITLE
Cb translate

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

## Proposed changes

- Live https://maehr.github.io/collectionbuilder-csv/

- See https://github.com/CollectionBuilder/cb-translate/pull/7
- See https://github.com/CollectionBuilder/cb-translate/issues/4
- See https://github.com/CollectionBuilder/cb-translate/issues/5

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **New Features**
	- Enhanced multilingual support across the site, dynamically fetching translated text for various elements based on the site's language setting.
	- Improved user experience by updating error messages and download link messages in the video player component.

- **Bug Fixes**
	- Fixed static error messages in the video player component to display dynamic, language-dependent messages for better user interaction.

- **Documentation**
	- Updated documentation to include details about the new multilingual support and dynamic content updates.

### CODE REVIEW

File: _includes/index/carousel.html
Comment (evanwill) [inline _includes/index/carousel.html:1]: This needs to keep the include text, like:
`{%- assign btn-text = include.btn-text | default: site.data.translations['_includes']['index']['carousel.html']['view-item'][site.lang] | default: "View Item" -%}`

**Comment by evanwill** [comment]
Wow @maehr this was a huge amount of work, thank you! It looks good. I think we should merge it, and keep working from there.

File: _includes/index/carousel.html
Comment (maehr) [reply _includes/index/carousel.html:1]: Thank you. I fixed it in https://github.com/CollectionBuilder/cb-translate/pull/10/commits/d3e794080b6b70f5e06bbd5c4a44714d7d038cd2

**Comment by maehr** [comment]
> Wow @maehr this was a huge amount of work, thank you! It looks good. I think we should merge it, and keep working from there.

Thanks for the review. It's been a lot of work, but I hope it's worth it.
PS: We are definitely having fun implementing new features in cb-csv over here <https://forschung.stadtgeschichtebasel.ch/>. It is really easy thanks to you guys.

**Pull request merged**